### DISCUSSION TITLE
Releases Updates and EOL

### DISCUSSION THREAD

**Discussion by yashsingh74** [coredns/coredns#6024] [General]
On the coredns website, there is no explicit information provided regarding upcoming release dates, end-of-life releases, or which releases are maintained by the community (LTS releases).

I'm not sure if this has been brought up before, but if not, could we keep the section up to date on the official CoreDNS page?

**Comment by chrisohaver** [comment]
I agree that we should add this info to the website somewhere.  

FWIW,

We don’t plan the release date of future releases in advance.

We only maintain the latest/current minor release. Currently this is 1.10.

**Reply by yashsingh74** [reply]
Should I create a PR and get started with something for the initial phase?

**Reply by chrisohaver** [reply]
Actually, I’ll just add it to the README in this repo. I don’t think it needs to also be on the website.

**Comment by humblec** [comment]
@yashsingh74 https://github.com/coredns/coredns/issues/5997
